roles/cinder-control/tasks/main.yml

```diff
@@ -53,6 +57,18 @@
     - "{{ cinder.services.cinder_api }}"
     - "{{ cinder.services.cinder_scheduler }}"
 
+- name: enable cinder encryption for barbican
+  template: src=etc/cinder/cinder.encryption.conf
+            dest=/etc/cinder/cinder.encryption.conf mode=0640
+            owner=root group=cinder
+  notify: restart cinder services
+  when: cinder.encryption.enabled|bool and barbican.enabled|default(false)|bool
```

Comment:
To my knowledge, we need this task either barbican is enabled or fixed_key enabled.

Please remove the `bool` filter, ansible can recognize it as bool